pkg/master/master.go

```diff
@@ -341,43 +335,35 @@ func (m *Master) InstallAPIs(c *Config) {
 }
 
 func (m *Master) initV1ResourcesStorage(c *Config) {
-	dbClient := func(resource string) storage.Interface { return c.StorageDestinations.Get("", resource) }
-	restOptions := func(resource string) generic.RESTOptions {
-		return generic.RESTOptions{
-			Storage:                 dbClient(resource),
-			Decorator:               m.StorageDecorator(),
-			DeleteCollectionWorkers: m.deleteCollectionWorkers,
-		}
-	}
 
-	podTemplateStorage := podtemplateetcd.NewREST(restOptions("podTemplates"))
+	podTemplateStorage := podtemplateetcd.NewREST(m.GetRESTOptionsOrDie(c, api.Resource("podTemplates")))
```

Comment:
> I still suggest maintaining the restOptions function for brevity/readability.

:(  Will do it, but I think it significantly reduces clarity for people scanning through.